discount/src/main/java/com/kyriba/conference/discount/DiscountApplication.java

```diff
@@ -0,0 +1,26 @@
+package com.kyriba.conference.discount;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@SpringBootApplication
+@EnableSwagger2
```

Comment:
Move to dedicated Swagger configuration in api package.

Reply:
done